conf/default.toml

```diff
@@ -106,6 +106,3 @@ background_color = "#fdfbff"
 #     Welcome to <span class="text-primary">Organization</span>'s<br>
 #     wonderful <span style="font-size: 1.2em;">data catalogue</span>
 #     """
-
-[translations.en]
-datahub.header.title.html = '<div class="text-white">Toutes les <span class="text-primary">données<br>publiques</span> de mon organisation</div>'
```

Comment:
+1 this has nothing to do in the default config, can't remember why we put it there!